packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -67,12 +73,12 @@ class CliServiceProvider implements ServiceProvider {
 
   async renameCollection(
     database: string,
-    collection: string,
+    oldName: string,
```

Comment:
this is actually a database kind-of method, didn't know how to communicate that, thought that probably renaming this parameter would help.